tests/ui/privacy/issue-113860-1.rs

```diff
@@ -0,0 +1,16 @@
+#![feature(staged_api)]
+//~^ module has missing stability attribute
```

Comment:
```suggestion
//~^ ERROR module has missing stability attribute
```
(And in other places.)

Reply:
Oops, fixed!